### DISCUSSION TITLE
What does "swipe to action" button to exactly?

### DISCUSSION THREAD

**Discussion by NikunjK8** [fast4x/RiMusic#1166] [Q&A]
![IMG_20240322_235052](https://github.com/fast4x/RiMusic/assets/99175083/61a4a250-7127-4101-a1ae-0c66e04dc8d3)

**Comment by OrangeZXZ** [comment]
> ![IMG_20240322_235052](https://github.com/fast4x/RiMusic/assets/99175083/61a4a250-7127-4101-a1ae-0c66e04dc8d3)
> 

it skips the song or gose to the previous song if you swipe on the image while listening

**Reply by NikunjK8** [reply]
But the same thing also works even if you disabled "swipe to action" and also have horizontal swipe enabled from player appearance. This is what confusing me.

**Reply by fast4x** [reply]
Swipe to action is a hidden menu under every song. It permit to select specific action, when disable it only an action is possible and is delete song with horizontal swipe gesture.

**Reply by NikunjK8** [reply]
Got it
